codecov.yml

```diff
@@ -0,0 +1,12 @@
+coverage:
+  status:
+    project:
+      default:
+        enabled: yes
+        target: auto
+    patch:
+      default:
+        enabled: yes
+        target: 100%
```

Comment:
@rlucioni There was an error in the config, so now it works properly - thank you for you help!